org.eclipse.winery.repository.ui/src/app/wineryTableModule/wineryTable.component.ts

```diff
@@ -251,6 +252,17 @@ export class WineryTableComponent implements OnInit, DoCheck {
     onCellClick(data: WineryRowData) {
         this.cellSelected.emit(data);
         this.currentSelected = data.row;
+        this.highlightSelectedRow(data.row);
+    }
+
+    private highlightSelectedRow(selectedRow: any): void {
+        const rowNumber: number = selectedRow ? this.rows.findIndex(row => row === selectedRow) : -1;
+        const tableRows = this.tableContainer.nativeElement.children[0].children[0].children[1].children;
```

Comment:
What happens, if you change the table site?

Reply:
![highlighting](https://user-images.githubusercontent.com/7428665/42556244-418e6d72-84eb-11e8-965b-d8c039690993.PNG)

Do you mean like this?

Reply:
It is still working the tableRows contains the rows visible in the current view.